modules/rostests/apitests/ntdll/NtAllocateVirtualMemory.c

```diff
@@ -738,7 +798,83 @@ CheckSomeDefaultAddresses(VOID)
     ok_ntstatus(Status, STATUS_SUCCESS);
 
 #ifdef _WIN64
-    skip("ToDo, 64-bit: Add 0x140000000/Exe and 0x180000000/DLL checks\n");
+    // 0x0000000140000000: 64-bit Exe image base.
```

Comment:
It would seem more useful to test "exe file is loaded at 140000000" and "trying to allocate memory where the exe file lives fails" separately. This is a test about the behavior of NtAllocateVirtualMemory, you're testing the address space layout of the process, which is just confusing and should have a separate test instead.